src/Control/Effect/Interpose.hs

```diff
@@ -18,10 +21,8 @@ import Control.Monad.Trans.Class
 -- the intercepted effect, and you can pass the effect on to the original handler
 -- using 'send'.
 --
--- The following shows the use of 'runInterpose' to eavesdrop on 'State' effects:
--- @
---  runM . runState @Int 5 . runInterpose @(State Int) (\op -> liftIO (print "State op") *> send op) $ go
--- @
+--   prop> run . evalState @Int a . runInterpose @(State Int) (\op -> modify @Int (+b) *> send op) $ modify @Int (+b) == a + b + b
```

Comment:
I’m not sure this is actually getting run, given the relative precedences of `==` and `$`. `doctest` doesn’t like telling you about errors, unfortunately 😞